cmd/projects/export.go

```diff
@@ -0,0 +1,42 @@
+package projects
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/tensorleap/leap-cli/pkg/entity"
+	"github.com/tensorleap/leap-cli/pkg/project"
+)
+
+func NewExportCmd() *cobra.Command {
```

Comment:
Even though the command is under projects package I think it should renamed to ExportProject

Reply:
This is the convention so far